main.go

```diff
@@ -31,8 +34,17 @@ import (
 )
 
 var (
-	scheme   = runtime.NewScheme()
-	setupLog = ctrl.Log.WithName("setup")
+	scheme                         = runtime.NewScheme()
+	setupLog                       = ctrl.Log.WithName("setup")
+	defaultAuthenticatorsAvailable = [...]string{"noop", "unauthorized", "anonymous", "oauth2_client_credentials", "oauth2_introspection", "jwt"}
+	defaultAuthorizersAvailable    = [...]string{"allow", "deny", "keto_engine_acp_ory"}
```

Comment:
I don't really like the duplication here - how about using JSON Schema instead? It would be similar to https://github.com/ory/oathkeeper/blob/master/.schemas/config.schema.json and we could even version it!